hathor/transaction/storage/migrations/add_feature_activation_bit_counts_metadata2.py

```diff
@@ -0,0 +1,41 @@
+#  Copyright 2023 Hathor Labs
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from typing import TYPE_CHECKING
+
+from structlog import get_logger
+
+from hathor.transaction.storage.migrations import BaseMigration
+from hathor.util import progress
+
+if TYPE_CHECKING:
+    from hathor.transaction.storage import TransactionStorage
+
+logger = get_logger()
+
+
+class Migration(BaseMigration):
```

Comment:
Should we add a description in the docstring? If I got it right, you skipped the previous migration because it could have already been run and you need to make sure it will run again. Is that correct?

Reply:
> If I got it right, you skipped the previous migration because it could have already been run and you need to make sure it will run again. Is that correct?

Exactly. In fact, all of this is going to be removed after your PR: https://github.com/HathorNetwork/hathor-core/pull/572#discussion_r1459639106

Reply:
Got it. Thanks!